src/redux/modules/volunteer.ts

```diff
@@ -10,6 +11,7 @@ const SELECT_VOLUNTEER = 'volunteer/SET_VOLUNTEER';
 const LOADING = 'volunteer/LOADING';
 const LOADING_DETAILS = 'volunteer/LOADING_DETAILS';
 const ERROR = 'volunteer/ERROR';
+const LOAD_DRAFTS = 'volunteer/LOAD_DRAFTS';
```

Comment:
Would cahnge it to SET_DRAFTS

Reply:
yes changed this and the function name as well